toonz/sources/tnztools/typetool.cpp

```diff
@@ -856,7 +856,11 @@ void TypeTool::updateTextBox() {
 void TypeTool::updateMouseCursor(const TPointD &pos) {
   int oldCursor = m_cursorId;
 
-  if (!m_validFonts)
+  TImageP img      = getImage(false);
+  TVectorImageP vi = img;
+  TToonzImageP ti  = img;
+
+  if (!m_validFonts || (!vi && !ti))
```

Comment:
For the Type tool, it seems that the modification done at line 425 is sufficient.
This tool is available on empty target but the condition here sets the cursor id to `CURSOR_NO` when selecting the empty cell.